portal-bridge/src/census.rs

```diff
@@ -161,6 +169,14 @@ impl Network {
     // peers. However, since the census continues to iterate through the peers after initialization,
     // the initialization is just to reach a critical mass of peers so that gossip can begin.
     async fn init(&mut self) {
+        let msg = match self.filter_clients.is_empty() {
+            true => format!("Initializing {} network census.", self.subnetwork),
+            false => format!(
+                "Initializing {} network census with filtered clients: {:?}",
+                self.subnetwork, self.filter_clients
+            ),
+        };
+        info!("{}", msg);
```

Comment:
I think it's slightly cleaner and clearer to drop the formats and log from inside the match.

```suggestion
        match self.filter_clients.is_empty() {
            true => info!("Initializing {} network census", self.subnetwork),
            false => info!(
                "Initializing {} network census with filtered clients: {:?}",
                self.subnetwork, self.filter_clients
            ),
        };
```